guix-science/packages/geant4.scm

```diff
@@ -0,0 +1,699 @@
+                  expat
+                  clhep-2.4.7.1
+                  python-2
+                  python-3.10
+                  perl
+                  tcsh
+                  qtbase-5
+                  libxmu
+                  libxt))
+    (native-inputs `(("G4NDL" ,g4ndl-4.7)
+                     ("G4EMLOW" ,g4emlow-8.5)
+                     ("G4PhotonEvaporation" ,photon-evaporation-5.7)
+                     ("G4RadioactiveDecay" ,radioactive-decay-5.6)
+                     ("G4PARTICLEXS" ,g4particlexs-4.0)
+                     ("G4PII" ,g4pii-1.3)
+                     ("G4RealSurface" ,real-surface-2.2)
+                     ("G4SAIDDATA" ,g4saiddata-2.0)
+                     ("G4ABLA" ,g4abla-3.3)
+                     ("G4INCL" ,g4incl-1.2)
+                     ("G4ENSDFSTATE" ,g4ensdfstate-2.3)
+                     ("G4TENDL" ,g4tendl-1.4)))))
+
+(define-public geant4-vis-11-2-1
+  (package
+    (inherit geant4-vis-11-2-0)
+    (version "11.2.1")
+    (source
+     (origin
+       (inherit (package-source geant4-vis-11-2-0))
+       (uri (git-reference
+             (url "https://gitlab.cern.ch/geant4/geant4")
+             (commit (string-append "v" version))))
+       (sha256
+        (base32 "1pdg5pf5izd2i05w34jclg0h4hrn3s4mjnr2dvq24lspjisc9r81"))))))
+
+;; -- dawn: DAWN visualisation tool for Geant4 -- 
+
+(define-public dawn
+  (package
+    (name "dawn")
+    (version "3.91a")
+    (source
+     (origin
+       (method url-fetch)
+       (uri "https://geant4.kek.jp/~tanaka/src/dawn_3_91a.tgz")
+       (sha256
+        (base32 "1x7mpi77jylsv8mzsqs0ppchbq147azd0b94i2qq2xhis7m5bn41"))))
+    (build-system gnu-build-system)
+    (native-inputs `(("gcc-toolchain" ,gcc-toolchain)
+                     ("make" ,gnu-make)))
+    (propagated-inputs `(("imagemagick" ,imagemagick)
+                         ("tk" ,tk)))
+    (arguments
+     `(#:make-flags (list (string-append "INSTALL_DIR=" %output "/bin"))
+       #:phases (modify-phases %standard-phases
+                  (replace 'configure
+                    (lambda* (#:key outputs #:allow-other-keys)
+                      #t))
+                  (replace 'check
+                    (lambda* (#:key outputs #:allow-other-keys)
+                      #t))
+                  (add-after 'build 'create-bin-directory
+                    (lambda* (#:key outputs #:allow-other-keys)
+                      (mkdir-p (string-append %output "/bin")) #t))
+                  (add-after 'unpack 'change-install-directory
+                    (lambda* (#:key outputs #:allow-other-keys)
+                      (substitute* "Makefile"
+                        (("/usr/local/bin")
+                         (string-append %output "/bin"))) #t)))))
+    (synopsis "Visualize 3D data generated by Geant4 simulation")
+    (description
+     "Fukui Renderer DAWN (Drawer for Academic WritiNgs) is a renderer,
+which reads 3D geometrical data and visualize them. It is a vectorized
```

Comment:
Please use two spaces after a period.  (It's an Emacs convention for navigating between sentences.)